dev-docs/bidders/nexverse.md

```diff
@@ -0,0 +1,72 @@
+
+---
+biddercode: nexverse
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+maintainer: support@nexverse.ai
+media_types:
+  - banner
```

Comment:
please remove the mediatypes from their own lines. They need to be concatenated as shown above